src/pages/MainPage.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, useState } from 'react';
+import ResultCatalog from '../components/result-catalog/ResultCatalog';
+import Header from '../components/header/Header';
+
+export const MainPage: FC = () => {
+  const [inputValue, setInputValue] = useState<string>(() => {
+    const storedValue = localStorage.getItem('inputValue');
+    return storedValue || '';
+  });
```

Comment:
Use an empty string as a default value for useState hook. Put the value from localStorage in a useEffect hook

Reply:
Thank's!!!